### ISSUE TITLE
Add `brotli` package

### ISSUE THREAD

**Issue by MuTsunTsai** [pyodide/pyodide#2921]
## 🐍 Package Request

- Package Name and Version : brotli 1.0.9
- Package URL : https://github.com/google/brotli, https://pypi.org/project/brotli
- Package Dependencies that needs to be resolved first: none

I'm new to pyodide and I followed the docs to use `pyodide_build mkpkg` (which is quite difficult to get it to work on Windows, by the way). It gives me the following meta.yaml:

```
package:
  name: brotli
  version: 1.0.9
source:
  url: https://files.pythonhosted.org/packages/2a/18/70c32fe9357f3eea18598b23aa9ed29b1711c3001835f7cf99a9818985d0/Brotli-1.0.9.zip
  sha256: 4d1b810aa0ed773f81dceda2cc7b403d01057458730e309856356d4ef4188438
test:
  imports:
  - brotli
about:
  home: https://github.com/google/brotli
  PyPI: https://pypi.org/project/brotli
  summary: Python bindings for the Brotli compression library
  license: MIT
```

I'm not sure if I should add anything. And I'm a bit confused that running `pyodide_build buildpkg` doesn't really generate anything other than an empty build.log file. Since I have no idea if I'm doing it right, I suppose I'll just put a request here. Thanks!

**Comment by MuTsunTsai** [comment]
OK I finally setup everything and successfully built `brotli` by following the latest (not the stable) version documentation. The .yaml file shown above built directly without any patches, and it creates this `Brotli-1.0.9-cp310-cp310-emscripten_3_1_14_wasm32.whl` file. Import the package in `console.html` also seems working. What should I do next? Should I just make a PR with the .yaml file?

**Comment by ryanking13** [comment]
Thanks for your work @MuTsunTsai!

> What should I do next? Should I just make a PR with the .yaml file?

Yes, please make a PR with .yaml file and some tests.

**Comment by MuTsunTsai** [comment]
@ryanking13 It's done but some help might be needed though. Could you take a look?

**Comment by ryanking13** [comment]
Closing as resolved #2925. Thanks @MuTsunTsai!
